### PR TITLE
create rare disease subset documentation

### PR DESCRIPTION
**Pull request by sabrinatoro**
@nicolevasilevsky and @matentzn please review this documentation.

(note that there is an image to add, but I am not able to add it on the desktop repo. I will try on the online one.)

### CODE REVIEW

File: docs/editors-guide/rare-disease-subset.md
Comment (sabrinatoro) [inline docs/editors-guide/rare-disease-subset.md:24]: note that this is not currently the case, but I will add a GH issue to update the sparql

File: docs/editors-guide/rare-disease-subset.md
Comment (sabrinatoro) [inline docs/editors-guide/rare-disease-subset.md:46]: still need to update the website link here

**Comment by sabrinatoro** [comment]
@nicolevasilevsky and @matentzn 
I updated the document based on my discussion with Nico during our 1:1.

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:9]:
```suggestion

![rareexample](https://github.com/monarch-initiative/mondo/assets/12737987/a5dc4b6e-c3d1-4bca-926a-f81fdcea5ddb)
```

Otherwise wrong line break.

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:8]:
```suggestion
Example: 'calciphylaxis' (MONDO:0017215) is considered "rare" according to NORD, GARD, and Orphanet, as shown by the 'in_subset' annotations 'nord_rare', 'gard_rare', 'orphanet_rare'.
```

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:13]: This is IMO too weak.. Is there any hope we can drop this phrase and replace it by the operational criteria that _led us_ to assert that axiom?

**Review by matentzn — COMMENTED** [review]
2 tiny, 1 medium suggestions.

File: docs/editors-guide/rare-disease-subset.md
Comment (sabrinatoro) [reply docs/editors-guide/rare-disease-subset.md:13]: We 'could' but these came before my time, and I don't know if there were any operational criteria... 

@nicolevasilevsky, do you know how these "bearer of rare" axiom were created? All I could find is that it was provided by the sources.

Comment (sabrinatoro) [reply docs/editors-guide/rare-disease-subset.md:14]: Note to self: can also add this reference: https://pubmed.ncbi.nlm.nih.gov/32020066/

**Comment by sabrinatoro** [comment]
@nicolevasilevsky and @matentzn 
New version of the document is ready. I added more information about where the "bearer of some rare" axioms came from.
We need to discuss these axioms (https://github.com/monarch-initiative/mondo/issues/6448), but I think it is good for now.

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:12]: You only list Orphanet here as the source for "bearer of some rare". Why then not keep this in sync with Orphanet? The only thing that would matter if Mondo curators would add this restriction on a class that was not defined as rare according to anyone else. INn that case, we need to say what Mondo considers rare at all.

Here is an idea: we query for all classes that are rare according to mondo but no one else (https://api.triplydb.com/s/urdlrUxt_), and develop a short narrative for why these should be considered rare despite no one else saying so? I don't think this is worth doing until _after_ there GARD and NORD PRs are merged though.

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:21]: This is currently not true. There are currently 1/3 of the classes only supported by Mondo!

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:36]:
```suggestion
Users can filter using the subset annotation “rare” to get a list of Mondo rare diseases without including their ontological parents.
```

Comment (matentzn) [reply docs/editors-guide/rare-disease-subset.md:36]: mondo_rare is only bearer of some rare

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:45]:
```suggestion
| GARD | [https://rarediseases.info.nih.gov/](https://rarediseases.info.nih.gov/) | gard_rare | *(coming soon)*|
```

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:46]:
```suggestion
| NORD | [https://rarediseases.org/](https://rarediseases.org/) | nord_rare | *(coming soon)*|
```

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:47]:
```suggestion
| Orphanet | [https://www.orpha.net/](https://www.orpha.net/) | orphanet_rare | *(coming soon)*|
```

**Review by matentzn — COMMENTED** [review]
We are getting close :) Great text, but I think we need to sort out the `mondo_rare` subset description better.

File: docs/editors-guide/rare-disease-subset.md
Comment (sabrinatoro) [reply docs/editors-guide/rare-disease-subset.md:12]: I don't know why we are not keeping it in sync with Orphanet... From my point of view, having "orphanet-rare" subset annotation and "bearer of rare" for orphanet is redundant (or would be redundant if we were keeping it in sync)
IMO, these axioms should be added to "every" rare diseases, or to only diseases that a curator identified as rare (and is not in a rare subset). 

I created an [issue](https://github.com/monarch-initiative/mondo/issues/6448) to discuss this.

**Comment by sabrinatoro** [comment]
@matentzn and @nicolevasilevsky 
Please see the updated document based on our discussions. There are still a few things to change (e.g. I need to update the image), but before I take care of these details, please review and let me know if things make more sense. 

@matentzn : did we completely remove all "bearer of rare" annotations, or are we automatically adding them to all rare subset Mondo terms? (I didn't see it in the "Construct" files, but maybe I missed it)

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:25]: Have you checked this with the subset report query? I seem to remember there were huge differences. We could also add an overlap plot here (venn diagram) to illustrate the overlap. You can make a separate issue if you would like that (not a blocker here).

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:40]: Missing link to obsolescence docs

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:42]:
```suggestion
1. If a MONDO term (MONDO id) is obsoleted, the Mondo term will not appear in the rare disease subset, regardless of wether the disease is considered rare by one of the authoritative rare disease sources. The rare disease authoritative source reporting this disease will be contacted to report this obsoletion and its reasons.
```

File: docs/editors-guide/rare-disease-subset.md
Comment (matentzn) [inline docs/editors-guide/rare-disease-subset.md:43]:
```suggestion
2. If an authoritative rare disease source obsoletes a disease term, or removes a disease from their rare disease list, the corresponding Mondo term will lost its corresponding "rare subset" annotation for this particular source. It should be noted that the rare subset annotation for the other authorities will not be modified unless they also obsolete or remove the corresponding disease from their rare disease list.
```

**Review by matentzn — COMMENTED** [review]
Absolutely fantastic! I made some minor comments that can be quickly fixed, it's ready to merge (no more review required by me).

File: docs/editors-guide/rare-disease-subset.md
Comment (sabrinatoro) [reply docs/editors-guide/rare-disease-subset.md:26]: I haven't compared the different subsets and I agree having a venn diagram would help. Let's talk about how to do this, because it is not necessarily straight forward. I will make a separate issue.

**Comment by sabrinatoro** [comment]
I think this is ready to merge? @nicolevasilevsky do you want to review first?

@matentzn : do you think we should merge this when we merge the [rare subset revision PR](https://github.com/monarch-initiative/mondo/pull/6506)?

**Review by matentzn — APPROVED** [review]
Nice work!

**Comment by matentzn** [comment]
No worries the subset PR is also ready. Can be merged now (both).

**Pull request merged**
